digital_asset_types/src/dao/asset.rs

```diff
@@ -97,8 +101,8 @@ impl ColumnTrait for Column {
             Self::RoyaltyTarget => ColumnType::Binary.def().null(),
             Self::RoyaltyAmount => ColumnType::Integer.def(),
             Self::ChainDataId => ColumnType::BigInteger.def().null(),
-            Self::CreatedAt => ColumnType::Timestamp.def().null(),
-            Self::BurntAt => ColumnType::Timestamp.def().null(),
+            Self::CreatedAt => ColumnType::TimestampWithTimeZone.def().null(),
```

Comment:
Do you want this to be with timezone or without?  When I generated it it defaulted to with timezone, not sure if you had chosen that vs utc and why but wanted to point it out in case it reverted by mistake.